Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -59,9 +59,11 @@ public MainWindow()
 
         private void OnCopy(object sender, ExecutedRoutedEventArgs e)
         {
-            if (QueryTextBox.SelectionLength == 0)
+            var result = _viewModel.Results.SelectedItem?.Result;
+            if (QueryTextBox.SelectionLength == 0 && result != null)
             {
-                _viewModel.ResultCopy(string.Empty);
+                string copyText = result.CopyText;
+                _viewModel.ResultCopy(copyText);
 
             }
             else if (!string.IsNullOrEmpty(QueryTextBox.Text))
```

Comment:
When copying `QueryTextBox` it pops up an notification. I think in this case just copy the text without checking if it's a file is enough.